### ISSUE TITLE
Telegram desktop not syncing messages across devices automatically?

### ISSUE THREAD

**Issue by marcopaganini** [telegramdesktop/tdesktop#4257]
### Steps to reproduce
1. Start telegram desktop (currently v1.2.1, but tested on older versions as well) on two computers in my network: Let's call them "desktop" and "laptop". Login as usual.
2. Go to the desktop computer, choose friend X, send, say, 3 messages 1, 2, and 3.
3. Friend responds to messages, response is shown. Everything is normal.
4. On the laptop, click on friend X. Nothing shows in the conversation.
5. Send one message to friend X on laptop. All messages previously sent on desktop (messages 1, 2, 3 and response) now show on laptop. Conversation proceeds normally on laptop.
6. Looking at the screens (desktop and laptop) side by side, I can see that while I'm exchanging messages on the laptop, the desktop receives responses sent by friend X, but not messages sent by me.
7. If I go back to the desktop and send anything to friend X from there, the conversation is synced again, and I'll see the same behavior happening on the laptop (laptop receives messages send by friend X, but not my own messages -- friend X receives all messages normally).

I have debugging logs but would have to comb them over for private information, or I could try to reproduce this in an easy way to have minimal logs that show the problem.

### Expected behaviour
All instances of telegram should be in sync. This is what happens between the Android client and telegram desktop for linux, for example.

### Actual behaviour
Telegram desktop (for Linux) appears to use the last instance where a message was sent as the only instance that receives message updates. After a certain amount of time, a client that hasn't received message updates won't recover, requiring a telegram restart (such as sleeping laptops).

### Configuration
- Mint Linux 17.1 (Ubuntu 14.04), kernel 3.13.0-128-generic

**Version of Telegram Desktop:**
1.2.1

**Used theme**:
No theme.

<details><summary><b>Logs</b>:</summary>
Insert logs here (if necessary)

</details>

**Comment by john-preston** [comment]
@marcopaganini Was it possible that you've copied application data from one laptop to another and use both instances of Telegram Desktop with one authorization?

**Comment by marcopaganini** [comment]
Bingo! I did copy the data directory when I first installed telegram a long time ago and I've been having these problems intermittently ever since. I removed the data directory and reinstalled. The problem apparently disappeared. Thanks!

BTW, configuring telegram for each instance is a pain. I wonder if we should have a feature request to save the configuration in the cloud, or at least export it in a way other instances can read.

I'm closing this bug. Thanks.
